### DISCUSSION TITLE
Separate notification channel for release announcements

### DISCUSSION THREAD

**Discussion by harry-xm** [fluent/fluent-bit#5160] [General]
Previously I'm subscribed to all releases in this repository, but now that nightly releases are made everyday there is too much noise to keep up. Could you set up a low-traffic mailing list for release announcements only?

**Comment by patrick-stephens** [comment]
Ah, sorry for that @harry-xiaomi - we're trying to get nightly builds in to automate things and make it easy to test plus have a regular release cadence #4875 

They should all be tagged as `unstable-*` and marked as `pre-release` but it looks like there is no way to filter this easily, I did find a few requests for it when I searched but no easy solution without parsing API responses yourself.

Would it be ok to add an announcement here in discussions for releases? @lecaros 
I could automate this on the release job once #4875 is completed.

**Reply by patrick-stephens** [reply]
I've also added an update to the feature request here: https://github.com/github/feedback/discussions/10776#discussioncomment-2521691

@harry-xiaomi and others would be useful to indicate demand for the feature there.

**Comment by seveas** [comment]
Another option could be to publish the pre-releases in a separate repository.

**Reply by patrick-stephens** [reply]
Yeah that's probably the best option thinking about it so I'll see if I can update things to do that.

https://github.com/fluent/fluent-bit/issues/5259

**Comment by patrick-stephens** [comment]
@harry-xiaomi I'm going to move them to https://github.com/fluent/fluent-bit-unstable-releases

**Reply by patrick-stephens** [reply]
This should be sorted now.
